modules/redpanda/redpanda.go

```diff
@@ -86,6 +86,13 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 				"--smp=1",
 				"--memory=1G",
 			},
+			WaitingFor: wait.ForAll(
+				// Wait for the ports to be exposed only as the container needs configuration
+				// before it will bind to the ports and be ready to serve requests.
+				wait.ForExposedPortOnly(defaultKafkaAPIPort),
+				wait.ForExposedPortOnly(defaultAdminAPIPort),
+				wait.ForExposedPortOnly(defaultSchemaRegistryPort),
```

Comment:
Failing because that function does not exist. See my comment here https://github.com/testcontainers/testcontainers-go/pull/2691#discussion_r1706718746

Reply:
Ah yes, depends on #2691